web/src/model/filters.ts

```diff
@@ -10,39 +10,28 @@ export enum FilterComponent {
 export enum FilterCategory {
   Source,
   Destination,
-  Common,
   None
 }
 
 export type FilterId =
-  | 'namespace'
   | 'src_namespace'
   | 'dst_namespace'
-  | 'name'
   | 'src_name'
   | 'dst_name'
-  | 'kind'
   | 'src_kind'
   | 'dst_kind'
-  | 'owner_name'
   | 'src_owner_name'
   | 'dst_owner_name'
-  | 'resource'
   | 'src_resource'
   | 'dst_resource'
-  | 'address'
   | 'src_address'
   | 'dst_address'
-  | 'mac'
   | 'src_mac'
   | 'dst_mac'
-  | 'port'
   | 'src_port'
   | 'dst_port'
-  | 'host_address'
   | 'src_host_address'
   | 'dst_host_address'
-  | 'host_name'
   | 'src_host_name'
   | 'dst_host_name'
   | 'protocol'
```

Comment:
Instead of managing each `src` / `dst` you can defined targeted filters separately and then build the final type as:
```
export type TargetedFilterId =
  | 'namespace'
  | 'name'
  | 'kind'
  | 'owner_name'
  | 'resource'
  | 'address'
  | 'mac'
  | 'port'
  | 'host_address'
  | 'host_name';

export type FilterId =
  | `src_${TargetedFilterId}`
  | `dst_${TargetedFilterId}`
  | 'protocol'
  | 'type'
  | 'id';
```